app/classifier/world_wide_telescope.cjsx

```diff
@@ -104,11 +104,26 @@ class Plate
 
     makeStarCoord = if @starChart.coordinateSystem() == StarChart.EQUATORIAL then StarCoord.fromRaDec else StarCoord.fromGlatGlon
     xAxisDec = if @starChart.xAxis.unit == Axis.DEC || @starChart.xAxis.unit == Axis.GLAT then true else false
+    @fullValues xRange
+    @fullValues yRange
     @coordCorners = [
       makeStarCoord(xRange[0].value, yRange[0].value, xAxisDec), makeStarCoord(xRange[1].value, yRange[0].value, xAxisDec),
       makeStarCoord(xRange[1].value, yRange[1].value, xAxisDec), makeStarCoord(xRange[0].value, yRange[1].value, xAxisDec)
     ]
 
```

Comment:
it's not really clear to me, except from the commit message, what this function is doing. it might need either a more descriptive name or a comment